<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileConvert_demo</class>
 <widget class="QDialog" name="FileConvert_demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>336</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>468</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileConvert_demo</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>541</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Source Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Dest Path:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="SourcePathEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="BrowseSourcePath">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="DestPathEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="BrowseDestPath">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="ConvertProgress">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>541</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="StartConvert">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopConvert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QRadioButton" name="radioBtnAVI">
      <property name="text">
       <string>AVI</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioBtnASF">
      <property name="text">
       <string>ASF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioBtnMP4">
      <property name="text">
       <string>MP4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioBtnTS">
      <property name="text">
       <string>TS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioBtnPS">
      <property name="text">
       <string>PS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioBtnDAV">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>DAV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_Seg"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_Seg_Size">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Sgmt Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Seg_Size"/>
      </item>
      <item>
       <widget class="QLabel" name="label_Seg_Units">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>MB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
